.github/workflows/linux.yml

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) 2023 Sebastian Pipping <sebastian@pipping.org>
+# Licensed under GPL v2 or later
+
+name: Build on Linux
+
+on:
+  pull_request:
+  push:
+  schedule:
+    - cron: '0 3 * * 5'  # Every Friday at 3am
```

Comment:
Unnecessary? If not, could use a rationale comment.

Reply:
@mpartel the idea is the following: Ideally, all movement — all upgrades — are under our control in a CI so that everything that worked yesterday still works the same today. In reality, there is always something moving without our control in GitHub Actions because the images — despite being based on LTS ones — change slighty from time to time and the CI is red and all pull request starting off `master` will be red to without being guilty — not ideal. To improve on that situation, the idea is to build at least once a week so that even in a period of time without pull requests or pushes for weeks we still notice when the ground below us moved out of all-fine land. What do you think?

Reply:
Ok, I was curious if you had a particular changing thing in mind. Anyway, since there's almost no downside, sure let's keep it.